js/layout-javascript/news-feed-code.js

```diff
@@ -221,6 +221,8 @@ DynamicList.prototype.attachObservers = function() {
         return;
       }
 
+      _this.$container.find('.news-feed-list-wrapper').hide();
```

Comment:
This looks like the main the list being toggled on and off. Why?

And when the list is shown again, does it restore its scroll position? e.g. if the list was previously scrolled to 30% down the list, does it restore to the same point?